src/main/java/org/minions/devfund/dennis/Squarelotron.java

```diff
@@ -0,0 +1,108 @@
+package org.minions.devfund.dennis;
+
+/**
+ * This is the methods for Squarelotron test.
+ */
+public class Squarelotron {
+    private int[][] squarelotronMain;
+    private int size;
+
+    /**
+     * Constructor for squarelotron, filling the matrix.
+     * @param n dimension of matrix
+     */
+    public Squarelotron(int n) {
+        this.size = n;
+        int numbers = 1;
+        squarelotronMain = new int[n][n];
+        for (int fil = 0; fil < n; fil++) {
+            for (int col = 0; col < n; col++) {
+                this.squarelotronMain[fil][col] = numbers;
+                numbers++;
+            }
+        }
+    }
+
+    /**
+     * This method found numbers in the the ring or not in ring.
+     * @param fil all files of the matrix
+     * @param col all columns of the matrix
+     * @param ring what ring you select
+     * @return this return numbers in the ring and not in the ring
+     */
+    public boolean numbersInRing(int fil, int col, int ring) {
```

Comment:
This method should be private
So it will not be necessary unit tests for that.
So you can remove those unit tests.